docs/usage.md

```diff
@@ -54,6 +55,15 @@ Sample files that can be used as illustrative references can be found in this re
 | id | int | Unique course ID |
 | name | str | User friendly course name |
 
+## `course_contents.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| section | str | Section name |
+| activity_name | str | Activity name |
+| lesson_page | str | Lesson page name |
+| content_id | str | Content ID |
```

Comment:
Minor: Would it be more consistent with other instances of `content_id` here if this were?:

```
| content_id | UUID | Content page UUID |
```